common/activity.mk

```diff
@@ -0,0 +1,23 @@
+INTERNAL_PORT := 80
+CONT_NAME := ${IMG_NAME}
+FILE_FLAG := ../flag
```

Comment:
I don't see the benefit of using this variable and not simply doing `cat ../flag`

Reply:
Yep, you're right. I will do the change.